### ISSUE TITLE
`Extensions` page in documentation is currently blank

### ISSUE THREAD

**Issue by paddyroddy** [glass-dev/glass#186]
More to raise awareness
https://glass.readthedocs.io/latest/manual/extensions.html

**Comment by ntessore** [comment]
Yes, this is a leftover from a time when GLASS was essentially just the core namespace package, and the functionality was spread out over separate packages. Users found that very confusing, so now almost all of the functionality is in the base GLASS package, and `glass.ext` is the namespace for extensions.

Packages such as `glass.ext.camb` with extra dependencies (e.g., CAMB) would still be extensions, so not sure if the page in the documentation is superfluous now.
